src/main/resources/hudson/plugins/im/IMPublisher/dropdownDescriptorSelector.jelly

```diff
@@ -47,7 +47,7 @@ THE SOFTWARE.
       <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
         selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}">
         <j:set var="instance" value="${current.descriptor==descriptor ? current : null}" />
-        <tr><td><input type="hidden" name="stapler-class" value="${descriptor.clazz.name}"/></td></tr>
+        <input type="hidden" name="stapler-class" value="${descriptor.clazz.name}"/>
```

Comment:
Is it currently working? You might want an invisibleEntry tag wrapping it
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/invisibleEntry.jelly

```suggestion
        <f:invisibleEntry><input type="hidden" name="stapler-class" value="${descriptor.clazz.name}"/></f:invisibleEntry>
```

Reply:
Sounds reasonable, and you know better than myself, thanks for the suggestion :)